### DISCUSSION TITLE
Minio Cache Feature

### DISCUSSION THREAD

**Discussion by tsohst** [minio/minio#15669] [Q&A] [answered]
Hey,

we would like to use the minio caching feature.
Documentation sounds like this is just a minio gateway feature?
Since the minio gateway will be deprecated, is there an alternative?

Our current plan would be to have an ssd per minio node to store the cache on it.

https://docs.min.io/docs/minio-disk-cache-guide.html

**Comment by klauspost** [comment, marked as answer]
Caching is rarely a benefit. It adds complexity and hard to debug scenarios and reduces scalability and availability. Instead we just make the server fast.

You can look into [tiering](
https://docs.min.io/minio/baremetal/lifecycle-management/lifecycle-management-overview.html#object-transition-tiering). 

This allows to keep incoming data on a hot tier. After a time the data is transitioned to a warn tier, but metadata remains on the hot tier.

It you would like an in-depth discussion of your options beyond what is documented write `hello@min.io` and we can help you.
